workflows/ampliseq.nf

```diff
@@ -355,7 +358,28 @@ workflow AMPLISEQ {
         ch_stats = DADA2_MERGE.out.dada2stats
     }
 
-
+    //
+    // MODULE : ASV post-clustering with VSEARCH
+    //
+    if (params.vsearch_cluster) {
+        ch_fasta_for_clustering = DADA2_MERGE.out.fasta
+            .map {
+                fasta ->
+                    def meta = [:]
+                    meta.id = "ASV_post_clustering"
+                    [ meta, fasta ] }
+        VSEARCH_CLUSTER ( ch_fasta_for_clustering )
+        ch_versions = ch_versions.mix(VSEARCH_CLUSTER.out.versions.ifEmpty(null))
+        FILTER_CLUSTERS ( VSEARCH_CLUSTER.out.clusters, DADA2_MERGE.out.asv )
+        ch_versions = ch_versions.mix(FILTER_CLUSTERS.out.versions.ifEmpty(null))
+        MERGE_STATS_CLUSTER ( ch_stats, FILTER_CLUSTERS.out.stats )
```

Comment:
I run your branch and found an oddity: The stats (collected in `results/overall_summary.tsv`) are always read counts per sample, but in the case here, its ASV per sample. You can keep the ASV stats table, but would it be possible to add here for consistency the `reads per sample` instead of `asv per sample`?

Reply:
I wanted to add something to the stats table to be more consistent with the other filtering steps, but it might be easier to not add anything to the stat table for this step.

Reply:
Yes, probably not add anything. But its nice that the asv count stats table is generated and in the results folder.